### PR TITLE
[ADD] helpdesk_motive

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Review by nelsonramirezs — APPROVED** [review]
LGTM

**Review by nelsonramirezs — APPROVED** [review]

**Review by bodedra — APPROVED** [review]
LGTM :+1: Added minor suggestion.

File: helpdesk_motive/views/helpdesk_ticket_motive.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <field name="res_model">helpdesk.ticket.motive</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <menuitem id="helpdesk_motive"
+              name="Motives"
+              parent="helpdesk_mgmt.helpdesk_ticket_config_main_menu"
+              action="action_helpdesk_ticket_motive_tree"
+              sequence="5"/>
```
Comment (bodedra) [inline helpdesk_motive/views/helpdesk_ticket_motive.xml]:
```suggestion
              sequence="7"/>
```

**Comment by marcelsavegnago** [comment]
@max3903 Can you give me an example of motive and type?

**Pull request merged**